### PR TITLE
[12.0][MIG] stock_picking_tier_validation: Migration to 12.0

### PR DESCRIPTION
**Pull request by kaynnan**

### CODE REVIEW

**Review by marcelsavegnago — APPROVED** [review]
LGTM

File: stock_picking_tier_validation/__manifest__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "name": "Stock Picking Tier Validation",
     "summary": "Extends the functionality of Transfers to "
     "support a tier validation process.",
-    "version": "14.0.1.0.1",
+    "version": "12.0.1.0.1",
```
Comment (rousseldenis) [inline stock_picking_tier_validation/__manifest__.py]: Don't forget to reset to '12.0.1.0.0'

File: stock_picking_tier_validation/i18n/stock_picking_tier_validation.pot [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* stock_picking_tier_validation
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
```
Comment (augustodinizl) [inline stock_picking_tier_validation/i18n/stock_picking_tier_validation.pot:7]: "Project-Id-Version: Odoo Server 14.0\n"
```suggestion
"Project-Id-Version: Odoo Server 12.0\n"
```

File: stock_picking_tier_validation/README.rst [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+* Ecosoft
+
+Contributors
+~~~~~~~~~~~~
+
+* `Ecosoft <http://ecosoft.co.th>`__:
+
+  * Pimolnat Suntian <pimolnats@ecosoft.co.th>
+
```
Comment (augustodinizl) [inline stock_picking_tier_validation/README.rst:95]: * `Escodoo <http://escodoo.com.bt>`__:

  * Kaynnan Lemes <kaynnan.lemes@escodoo.com.br>

**Review by augustodinizl — APPROVED** [review]
LGTM - with comments 👍 

Good Job

File: stock_picking_tier_validation/views/stock_picking_views.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_picking_internal_search" model="ir.ui.view">
+        <field name="name">stock.picking.internal.search</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_internal_search" />
+        <field name="arch" type="xml">
```
Comment (augustodinizl) [inline stock_picking_tier_validation/views/stock_picking_views.xml:67]: it was aproved by me before, BUT...

i think that you missed that the buttons are declared in the view at v12...
look about it here (example): https://github.com/OCA/purchase-workflow/blob/12.0/purchase_tier_validation/views/purchase_order_view.xml

**Review by augustodinizl — CHANGES_REQUESTED** [review]

**Comment by kaynnan** [comment]
@augustodinizl Implemented the suggestions. I await your review again, please.

**Review by augustodinizl — APPROVED** [review]
Functional tests done, i think that it's ok now...

Congratulations!
success
!!
success 2 reviewers -rules
success 2 reviewers 2
success 2 reviewers

**Comment by augustodinizl** [comment]
@rousseldenis , i think it's ready to merge now, any considerations?

**Comment by marcelsavegnago** [comment]
ping @rousseldenis

**Comment by rousseldenis** [comment]
@kaynnan Do you want to keep commits as is ?

**Comment by kaynnan** [comment]
> @kaynnan Do you want to keep commits as is ?

Yes, I want to keep it as it is.

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**